configuration/programs/programs.csv

```diff
@@ -0,0 +1,3 @@
+Uuid,Void/Retire,Program concept,Outcomes concept,_order:1001
+5789a170-c020-4879-ae39-06b1de26cb5f,,ec41264d-e82e-4356-a7d2-7c1ff6c90abe,HIV Outcomes,
+b968ecad-8451-4193-af0c-dd0072c911a9,,477587a8-782c-43e6-ab77-1b69a36e200e,TB Program Outcomes,
```

Comment:
Let's try to be consistent:
"HIV Program Outcomes" sounds like the right label.

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/12/commits/d2c252fbcf28bb79577fc1794490296e5831f141)